From 8270abd84997458e0c0f89957e0fab47989366ad Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 28 May 2017 12:58:46 +0300
Subject: [PATCH 0657/1200] arm64: dts: armada-8040-mcbin: set cp1 comphy to
 SGMII2.5 mode

Currently, cp1 comphy5 is set to SGMII3 - this is incorrect.
According to mcbin board layout, Comphy5 should be set to
SGMII2 2.5G.

Change-Id: I565e65bb77a4dcb198ebf2a955eae152b8570dde
Signed-off-by: Igal Liberman <igall@marvell.com>
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55205
---
 arch/arm/dts/armada-8040-mcbin.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/armada-8040-mcbin.dts b/arch/arm/dts/armada-8040-mcbin.dts
index b6a4fe86f3..31698c3fbe 100644
--- a/arch/arm/dts/armada-8040-mcbin.dts
+++ b/arch/arm/dts/armada-8040-mcbin.dts
@@ -288,6 +288,7 @@
 		phy-type = <COMPHY_TYPE_SFI>;
 	};
 	phy5 {
-		phy-type = <COMPHY_TYPE_SGMII3>;
+		phy-type = <COMPHY_TYPE_SGMII2>;
+		phy-speed = <COMPHY_SPEED_3_125G>;
 	};
 };
-- 
2.22.0

